lib/common.sh

```diff
@@ -512,9 +514,11 @@ iterate(){
       break
     fi
     sleep "${TEST_TIME_INTERVAL}"
+    set +e
     # shellcheck disable=SC2068
     TMP_RET="$(${COMMAND})"
     TMP_RET_CODE="$?"
+    set -e
   done
   FAILS="$((FAILS+TMP_RET_CODE))"
```

Comment:
Were counting failures but not actually using it for anything, also also counting it based on the return value which can be anything else than 0 or 1 depending on the command?